src/modules/FSXAProxyApi.spec.ts

```diff
@@ -13,7 +13,7 @@ describe('FSXAProxyAPI', () => {
   beforeEach(() => {
     fetchMock.resetMocks()
     id = Faker.datatype.uuid()
-    locale = Faker.locale
+    locale = `${Faker.locale}_${Faker.locale.toUpperCase()}`
```

Comment:
I'm not sure if that's enough. Isn't en_EN an invalid locale? Wouldn't this generate something like en_DE or jp_GB?

Reply:
Semantically yes. But the fsxa stack does not semantically validate locales. In fact, "invalid" locales are explicitly supported and used by customers.
Whats important is the form. It always has to be "xx_YY"